index.js

```diff
@@ -116,13 +124,13 @@ function createQuestions(config) {
  * @return {String} Formated git commit message
  */
 function format(answers) {
-  const scope = answers.scope ? '(' + answers.scope.trim() + ') ' : ''
+  const scope = answers.scope ? '(' + answers.scope.trim() + ') ' : ' '
   const issues = answers.issues
     ? 'Closes ' + (answers.issues.match(/#\d+/g) || []).join(', closes ')
-    : ''
+    : ' '
 
   const head = truncate(answers.type + ' ' + scope + answers.subject.trim(), 100)
-  const body = wrap(answers.body || '', 100)
```

Comment:
Just to be sure about `''` → `' ' `, is that a code formatter issue or was it on purpose?
I'm asking because this is needed below by `filter(Boolean)` in order to strip empty lines.

Reply:
Right, I initially added " " to scope to get some breathing room in the header, but must have gotten trigger happy on the other lines as well. Will fix and update PR